src/lib/Theme.ts

```diff
@@ -1680,20 +1679,22 @@ export default class Theme {
     };
 
     private static getImageCdnBaseUrl = async () => {
-        let imageCdnUrl = '';
         try {
-            let startData = {
-                file_name: 'test.jpg',
-                content_type: 'image/jpeg',
-                size: '1',
-            };
-            let startAssetData = (
-                await UploadService.startUpload(
```

Comment:
We can remove `startUpload` method from `UploadService` file if not in use anywhere across project.

Reply:
Removed